src/SchemaNameBase.php

```diff
@@ -142,9 +141,7 @@ public function getElement(array $options = array()) {
         'content' => static::outputValue($value),
       ],
     ];
-    return array(
-      '#attached' => array('backdrop_add_html_head' => array(array($element, $id))),
-    );
+    return $element;
```

Comment:
In my testing, if you needed to add an HTTP header in addition to the HTML, you could do the following:

```
      $element['#attached']['backdrop_add_http_header'][] = array('X-My-Tag', static::outputValue($value)), TRUE);
```

But if you only need to add an HTML tag, there's no need to use `#attached` or `backdrop_add_html_head` at all.

Reply:
Excellent! Thanks for checking. I've tested, and it works well without the need for `#attached`.  Merging...